lib/python/treadmill_aws/cli/admin/cell/configure_data.py

```diff
@@ -59,12 +61,20 @@ def configure_data_cmd(docker_registries,
                            instance_profile,
                            subnets,
                            s3_registry_region,
-                           s3_registry_bucket):
+                           s3_registry_bucket,
+                           autoscale_min,
+                           autoscale_max):
         """Configure cell data."""
         admin_cell = admin.Cell(context.GLOBAL.ldap.conn)
         cell = admin_cell.get(context.GLOBAL.cell)
         data = cell.get('data', {})
 
+        autoscale = data.get('autoscale', {})
```

Comment:
Isn't this cell data? Autoscale configuration is per partition (so similar thing, but partition data).

Reply:
Assuming the change to move this data from cell to partition is merged... Michael, it is correct, should be modifying partition data. Do we have cli for managing partition data?

Reply:
It is merged. We have a CLI to set partition data in treadmill (treadmill admin ldap partition configure <partition> --data). Elegant version in treadmill-aws would be welcome (similar to configure_data for a cell), but partition can be configured as is.

Reply:
Moved the autoscale code to a new CLI command, configure_partition. 
We may have a case to put the autoscale code in both locations- there is no _default partition in LDAP. Having a "global" variable in the cell data would provide a fallback for any partitions that have not explicitly set variables of their own, as well as the _default partition. Thoughts?

Reply:
There is nothing preventing you from configuring _default partition in LDAP - _default partition doesn't need to be explicitly configured, but if it needs any configuration (like autoscale) then it can be configured. Code looks fine.